app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <h1 class="recipe-header">Recipes</h1>
+
+  <div id="recipes">
+    <% if @recipes.empty? %>
+      <p class="recipe-paragraph">No recipes available! </p>
+    <% else %>
+      <% @recipes.each do |recipe| %>
+        <div class="a">
+          <div class="one">
+            <%= recipe.name %>
+            <% if can? :delete, recipe %>
+              <%= button_to "Destroy this recipe", recipe, method: :delete %>
+            <% end %>
+            <p>
+            <%= link_to "Show", recipe %>
+          </p>
+            </div>
+            
+          <div>
+            <p><%= recipe.description %></p>
+          </div>
+          
+    </div>
```

Comment:
- [ Required] I think it would be better if you implement functionality to remove a `recipe` from the list. Currently, this is missing from your project yet it is a project requirement. Please see the screenshot below and rectify this.

![Screenshot from 2023-07-01 14-12-07](https://github.com/calebchris000/Recipe-App/assets/16392046/6648b194-9c84-4c10-a6a4-33c068f8039c)